features/networking/sgw-lgw-migration.feature

```diff
@@ -0,0 +1,55 @@
+Feature: SGW<->LGW migration related scenarios
+  
+  # @author anusaxen@redhat.com
+  # @case_id OCP-47561
+  @4.10
+  @admin
+  @destructive
+  @baremetal-upi @aws-ipi
```

Comment:
`  @network-ovnkubernetes` tag?

Do we also still use

```
    Given the env is using "OVNKubernetes" networkType
```

Reply:
Yea no harm in adding that. Though the test cases will be picked by network plugin defined in test cases. Thanks I added that as well.